### ISSUE TITLE
Showing pods log in version 2.4.1

### ISSUE THREAD

**Issue by klamas1** [harbur/kubernetic#117]
The new version 2.4.1 are no longer displayed pods log.

![image](https://user-images.githubusercontent.com/16032928/67065725-d0a79280-f188-11e9-8023-2aa1de9a57a8.png)

But in fact, there are logs:
```
# kubectl logs --since=10m pod/rabbitmq-dev-0 -c rabbitmq
2019-10-18 04:17:38.885 [info] <0.1802.3> accepting AMQP connection <0.1802.3> (10.233.96.12:38894 -> 10.233.96.8:5672)
2019-10-18 04:17:38.897 [info] <0.1802.3> connection <0.1802.3> (10.233.96.12:38894 -> 10.233.96.8:5672): user 'user' authenticated and granted access to vhost '/'
2019-10-18 04:17:39.261 [info] <0.1812.3> accepting AMQP connection <0.1812.3> (10.233.96.12:38916 -> 10.233.96.8:5672)
2019-10-18 04:17:39.270 [info] <0.1812.3> connection <0.1812.3> (10.233.96.12:38916 -> 10.233.96.8:5672): user 'user' authenticated and granted access to vhost '/'
2019-10-18 04:17:39.310 [info] <0.1822.3> accepting AMQP connection <0.1822.3> (10.233.96.12:38926 -> 10.233.96.8:5672)
2019-10-18 04:17:39.319 [info] <0.1822.3> connection <0.1822.3> (10.233.96.12:38926 -> 10.233.96.8:5672): user 'user' authenticated and granted access to vhost '/'
2019-10-18 04:17:39.672 [info] <0.1832.3> accepting AMQP connection <0.1832.3> (10.233.96.12:38952 -> 10.233.96.8:5672)
2019-10-18 04:17:39.678 [info] <0.1832.3> connection <0.1832.3> (10.233.96.12:38952 -> 10.233.96.8:5672): user 'user' authenticated and granted access to vhost '/'
```

**Comment by MichaelSp** [comment]
Further more, the app is crashing if I open the pod logs

**Comment by MichaelSp** [comment]
Is there a way to downgrade? I cant find the previous version....

**Comment by vasili439** [comment]
@MichaelSp agree with you. This release is terrible. A lot of cosmetic bugs which make Kubernetic useless (I'm using it since Apr 2018 and I know what I'm talking about). You can downgrade simply with modify download URL: https://kubernetic.s3.amazonaws.com/Kubernetic+Setup+2.3.0.exe

BTW: same issue with pod logs. + when you deleting pod it disappears from cluster but not from kubernetic.

**Comment by MichaelSp** [comment]
Thanks @vasili439 
btw: here is the mac download link https://kubernetic.s3.amazonaws.com/Kubernetic-2.3.0.dmg

**Comment by schuettecarsten** [comment]
Any news on this?

**Comment by KevinDenys** [comment]
I found a way to show the logs on mac using kubectx https://github.com/harbur/kubernetic/issues/127

**Comment by dkapanidis** [comment]
Log issues are now resolved on v2.5.0

Moreover logs can now be shown in controller resources (e.g. deployment logs) and be filtered by containers or pods:

<img width="818" alt="Screenshot 2020-03-02 at 21 32 51" src="https://user-images.githubusercontent.com/367397/75715322-835a3680-5ccd-11ea-93c9-5ded65cd385b.png">
